app/controllers/v1/users_controller.rb

```diff
@@ -62,6 +62,11 @@ def confirm_logging_code
     end
   end
 
+  def confirm_terms
```

Comment:
We need to verify somehow if this is user who has access to this account

Reply:
It is the same as in verification code. After logging in he is directly moved to this screen so in my opinion is fine to leave it as it is

Reply:
From my perspective, however, this is a slightly different behavior. In the case of 2FA, the verification code acts as a kind of password, in addition, it is valid for 15 minutes. So if someone has it and we can be sure that he is who he claims to be. In this our case, you don't have it.